mmv1/products/apigee/KeystoreAliasesSelfsigned.yaml

```diff
@@ -0,0 +1,192 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+    name: 'KeystoreAliasesSelfSigned'
```

Comment:
I think everything from here down is now two indentation levels (four spaces) to the right of where it should be.